EMIP/1001-2000/EMIP01975.xml

```diff
@@ -57,6 +65,60 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         </extent>
                      </supportDesc>
                   </objectDesc>
+                  <handDesc>
+                  <handNote xml:id="h1" script="Ethiopic">
+                  <seg type="rubrication">
+                  <foreign xml:lang="gez">ኢግዚአብሔር፡</foreign>
+                  The word God is rubricated in the first three works
+                  <foreign xml:lang="gez">ማርያም፡</foreign>
+                  the word Mary is rubricated in the final two works
+                  </seg>
+                  <desc>Columetric layout of text: <locus target="#124v #125r"/> (Ps. 150); <locus target="#135r #135v"/> (tenth Biblical Canticle)</desc>
+                  </handNote>
+                  </handDesc>
+                  <decoDesc>
+                  <decoNote type="frame" xml:id="d01">
+                    Decorative designs:
+                    <locus target="#98v #110r #113r #119r #125v #152v #154r #155r #156r"/> (haräg);
+                    <locus target="#24v"/> (black dots);
+                    <locus target="#44r #50v #60v #80v #86r"/> (red and black dots)
+                  </decoNote>
+                  </decoDesc>
+                  <additions>
+                  <list>
+                    <item xml:id="e01">
+                    <desc>
+                    <locus target="#ir #1v"/>: blank
+                    </desc>
+                    </item>
+                    <item xml:id="e02">
+                    <desc>
+                    Identified in the insurance hand list as Psalter 10
+                    </desc>
+                    </item>
+                    <item xml:id="e03">
+                    <desc>
+                    Navigation system:
+                    purple yarn is sewn into the fore edge of <locus target="#16 #36 #44 #72 #119 #138"/>;
```

Comment:
We describe it as leaf string marker (/leaf tab marker), under binding.  
https://betamasaheft.eu/Guidelines/?q=string&id=bindingDescription